src/js/modules/utilities/Logging.ts

```diff
@@ -9,40 +9,99 @@
  */
 
 import * as winston from "winston";
+import { Container, Logger, LoggerOptions } from "winston";
 
-export function newLogger(
-  module_name: string,
-  logLevel = "info",
-  simplify = false
-) {
-  const rpWasmFormatter = () => {
-    const logLineFormatter = winston.format.printf(
-      ({ level, message, label, timestamp }) => {
-        return `${timestamp} [${label}] ${level}: ${message}`;
-      }
-    );
-    const simpleLogLineFormatter = winston.format.printf(
-      ({ level, message, label }) => {
-        return `[${label}] ${level}: ${message}`;
-      }
-    );
+const maxLogFileSize = 5000000;
+
+const rpWasmFormatter = (moduleName: string, simplify: boolean) => {
```

Comment:
If your not ever setting 'simplify' you can remove it. I added it there so that in tests we wouldn't be adding the timestamp to the logs to make it easier to compare a static string to expected log data

Reply:
yes, we used it, for that reason, I don't remove it, I think is a good option for another log